ci/jobs-dev/run_post_mpas_hfip_HERA.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/sh
+
+#SBATCH -o out.post.mpas_hfip
+#SBATCH -e out.post.mpas_hfip
+#SBATCH -J mpas_hfip_test 
+#SBATCH -t 00:30:00
+#SBATCH --ntasks=256
+#SBATCH --cpus-per-task=4
+#SBATCH --time=00:30:00
+#SBATCH -q batch
+#SBATCH -A ovp
+#SBATCH --exclusive
+#SBATCH --mem=0
+
+
+set -x
+
+# specify computation resource
+export threads=1
+export MP_LABELIO=yes
+export OMP_NUM_THREADS=$threads
+export APRUN="srun"
+
+############################################
+# Loading module
+############################################
+module purge
+. $MODULESHOME/init/sh
+module use /contrib/spack-stack/spack-stack-1.8.0/envs/ue-intel-2021.5.0/install/modulefiles/Core
+module load stack-intel/2021.5.0
+module load stack-intel-oneapi-mpi/2021.5.1
+module load libpng/1.6.37
+module load jasper/2.0.32
+module load prod_util/2.1.1
+module load crtm/2.4.0.1
+module load wgrib2/3.1.1
```

Comment:
@SamuelTrahanNOAA This should be updated to spack-stack 1.9.1. That might be the source of the Hera issue. 
```suggestion
module use ${svndir}/modulefiles
module load hera_intel
module load prod_util/2.1.1
module load wgrib2/3.6.0
```